src/world/area_mac/mac_06/mac_06_3.c

```diff
@@ -28,14 +24,11 @@ NpcSettings N(NpcSettings_JrTroopa) = {
 };
 
 f32 D_80243434_867F74 = 0.0f;
-
-s32 D_80243438_867F78 = 0;
-
+f32 D_80243438_867F78 = 0.0f;
 s32 D_8024343C_867F7C = 0;
-
 s32 D_80243440_867F80 = 0;
 
-s32 func_80240E80_8659C0(Evt* script, s32 isInitialCall) {
+API_CALLABLE(N(func_80240E80_8659C0)) {
```

Comment:
Honestly unsure about this macro as it can hide the argument names

Reply:
Fair concern. Once youre aware of it, I think the macro makes it easier to visually scan a file and easier to search for all API-callable funcs within a certain set of files. It's self-documenting and enforces uniformity. I've already come across a dozen instances where the args were incorrect -- either spelled wrong, missing isInitialCall, or just void. That said, if we decide against it, we an always find-replace to get rid of it. For now, it saves on typing while the work is being done.